dashboard/components/cloud-account/components/CloudAccountsLayout.tsx

```diff
@@ -81,6 +82,15 @@ function CloudAccountsLayout({
               })}
           </div>
         )}
+        <div className="flex flex-col justify-end absolute bottom-10 border-t border-gray-300 p-4">
+          <Button
+            onClick={() => {
+              router.push('/onboarding/choose-cloud/');
+            }}
+          >
+            Add An Account
```

Comment:
```suggestion
            Add Account
```
@AvineshTripathi @mlabouardy  WDYT?

Reply:
or "Connect account" to keep it simple